### DISCUSSION TITLE
can't find copilot intellij plugin

### DISCUSSION THREAD

**Discussion by odamilola36** [community/community#7340] [Copilot Conversations] [answered]
I just got copilot and I can't find plugin on intelliJ

**Comment by linux-china** [comment, marked as answer]
I think you should upgrade your IntelliJ IDEA.   Copilot plugin is just for IntelliJ 2021.2 — 2021.3 (eap) only.

**Reply by jansorg** [reply]
Yes, thats's correct. @odamilola36 Please post which IDE and version you're using if you think you're using the right version.
